Gemfile

```
@@ -9,4 +9,4 @@ group :development do
 end
 
 gem "pry", group: [:development, :test], require: false
-gem "rails", "~> 6.1.7.3", group: [:development, :test], require: false
+gem "rails", "~> 6.1.7.4"
```

Lets keep the group here.

```suggestion
gem "rails", "~> 6.1.7.4", group: [:development, :test], require: false
```

---

Good catch